idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/kettlereboiler.py

```diff
@@ -0,0 +1,367 @@
+"""
+IDAES Kettle Reboiler
+
+Author : Paul Akula
+"""
+
+# Import Pyomo libraries
+from pyomo.environ import Expression, Constraint, value, Var, units
+from pyomo.common.config import ConfigBlock, ConfigValue, Bool
+
+# Import IDAES cores
+from idaes.core import (ControlVolume0DBlock,
+                        declare_process_block_class,
+                        MaterialBalanceType,
+                        MomentumBalanceType,
+                        UnitModelBlockData,
+                        useDefault)
+from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.util import get_solver
+import idaes.logger as idaeslog
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+@declare_process_block_class("KettleReboiler")
+class KettleReboilerData(UnitModelBlockData):
+    """Kettle Reboiler Unit Model."""
+
+    CONFIG = UnitModelBlockData.CONFIG()
+
+    # Configuration template for Phase specific  arguments
+    _PhaseCONFIG = ConfigBlock()
+
+    CONFIG.declare("specify_heat_duty", ConfigValue(
+        default=True,
+        domain=Bool,
+        description="Indicates if reboiler duty is specified",
+        doc="""Indicates if the user specifies the reboiler duty or not
+       **default** - True.
+        **Valid values:** {
+        **True** - heat duty is specified,
+        **False** - heat duty is not specified}"""))
+
+    CONFIG.declare("heat_duty", ConfigValue(
+        default=430.61,
+        domain=float,
+        description="Reboiler heat duty in kW",
+        doc="Reboiler heat duty in kW"))
+
+    _PhaseCONFIG.declare("property_package", ConfigValue(
+        default=useDefault,
+        domain=is_physical_parameter_block,
+        description="Property package to use ",
+        doc="""Property parameter object used to define property calculations
+        **default** - useDefault.
+        **Valid values:** {
+        **useDefault** - use default package from parent model or flowsheet,
+        **PhysicalParameterObject** - a PhysicalParameterBlock object.}"""))
+
+    _PhaseCONFIG.declare("property_package_args", ConfigBlock(
+        implicit=True,
+        description="Arguments to use for constructing property package",
+        doc="""A ConfigBlock with arguments to be passed to
+        property block(s) and used when constructing these,
+        **default** - None.
+        **Valid values:** {
+        see property package for documentation.}"""))
+
+    # Create individual config blocks for vapor and liquid phases
+    CONFIG.declare("vapor_side",
+                   _PhaseCONFIG(doc="Vapor phase config arguments"))
+    CONFIG.declare("liquid_side",
+                   _PhaseCONFIG(doc="liquid phase config arguments"))
+
+    def build(self):
+        # Call UnitModel.build to setup model
+        super(KettleReboilerData, self).build()
+
+        # Build Control Volume for liquid Phase
+        self.liquid_phase = ControlVolume0DBlock(default={
+            "dynamic": False,
```

Comment:
If this is a steady-state only model, then you should be over-riding the standard `dynamic` and `has_holdup` config arguments - see the EquilibriumReactor for an example.